pkg/system/reconciler.go

```diff
@@ -77,6 +77,8 @@ type Reconciler struct {
 	AWSCloudCreds       *cloudcredsv1.CredentialsRequest
 	AzureCloudCreds     *cloudcredsv1.CredentialsRequest
 	AzureContainerCreds *corev1.Secret
+	GCPContainerCreds   *corev1.Secret
```

Comment:
Please change the `GCPContainerCreds` to `GCPBucketCreds`.
Azure uses Containers but Google uses Buckets.
Make sure to change that in all of the occurrences in code.
Thank you.

Reply:
I see, will do. I followed the naming convention of other platforms. Thanks for the suggestion, I will make the changes accordingly.

Reply:
Done!